ruby_on_rails/advanced_forms_and_activerecord/installing_psql.md

```diff
@@ -112,6 +112,8 @@ You can try to run `psql` now, but you will get an error that the database does
 sudo -i -u postgres createdb <linux_username>
 ~~~
 
+*note: If your username has any capital letters, you must surround it in quotes when running the above command*
```

Comment:
Just to bring extra attention to these notes, we could use our note box styling:

```suggestion
<div class="lesson-note" markdown="1">
 If your username has any capital letters, you must surround it in quotes when running the above command*
</div>
```

I wonder if we should include these notes above the commands incase they are only seen after users run the command? But I'm really not sure if it will matter too much if the notes are already highlighted enough. Not a blocker, whatever you think is best works for me.

<img width="828" alt="Screenshot 2023-03-04 at 12 00 19" src="https://user-images.githubusercontent.com/7963776/222899979-7973963b-89f2-44fd-b8d9-af796acc008b.png">

Reply:
Both of those ideas are great I think! I'll make those changes

Reply:
I added the changes in. One thing I was struck by is that the two different sections preview with different gap styles between the note block and the code block below it.

From the linux section:
![linux_section_note](https://user-images.githubusercontent.com/88392688/222918729-f7048e0d-a198-40f0-8b79-233773e0ec26.png)

From the MacOS section:
![mac_section_note](https://user-images.githubusercontent.com/88392688/222918745-5c383e66-a2dd-4836-868a-8034429c73d5.png)

From using the inspector and experimenting, it seems to be caused by the linux section have the `bash` tag on the code block, which is giving it less of a top margin compared to the MacOS section, which uses a generic code block (that code is run in the context of pgsql service, so it wouldn't be semantically correct to call it bash). Just something I thought you might want to be aware of @KevinMulhern -- I personally think the MacOS section's styling looks slightly better but not a big deal or anything.

Reply:
Thanks for pointing this out @JoshDevHub. I agree the Mac OS spacing looks much better and should be the version we make everything consistent with. 

It should hopefully just require a little tweaking in our [code syntax custom styling file](https://github.com/TheOdinProject/theodinproject/blob/main/app/javascript/stylesheets/prism-theme.css). Would you like to make an issue in the site repo about it?

Reply:
Sure thing. I'll make the issue on the site repo :+1: